Edición de los requisitos funcionales y no funcionales para el proyecto.

LuisGilDzib opened the pull request:
He editado los requisitos funcionales y no funcionales a los necesarios para el proyecto, en espera de sugerencias o implementación en el proyecto

LuisGilDzib committed:
Create Prueba pull request

jaimeo13 committed:
Merge pull request #26 from ricardo-andre-pool-cen/Primera_Entrega

Actualización de ramas.

LuisGilDzib committed:
Create Requisitos funcionales

Aqui esta los requisitos funcionales, contribución

LuisGilDzib committed:
Update Requisitos funcionales

LuisGilDzib committed:
Update Requisitos funcionales

me falta editar los requisitos a cual nos sirven

LuisGilDzib committed:
Update Act#4 Requisitos no funcionales

Me falta editar los requisitos para generalizarlos, estos son una guía para los requisitos

ricardo-andre-pool-cen commented on Act#4 Requisitos no funcionales:
```diff
@@ -1,2 +1,42 @@
-Enlace al pizarron generado para esta actividad:
-https://www.canva.com/design/DAGQdZSTcDY/HYnIcSsqgOUo3Ea6Kvqyig/view?utm_content=DAGQdZSTcDY&utm_campaign=designshare&utm_medium=link&utm_source=editor
+Requisitos no funcionales.
+•	El sistema debe ser capaz de procesar N transacciones por segundo. Esto se medirá por medio de la herramienta X aplicada al “Software Testing” de servicios web.
```
No se especifica rendimiento a un prototipo no funcional, estos tipos de requerimientos no van

ricardo-andre-pool-cen commented on Act#4 Requisitos no funcionales:
```diff
@@ -1,2 +1,42 @@
+Requisitos no funcionales.
+•	El sistema debe ser capaz de procesar N transacciones por segundo. Esto se medirá por medio de la herramienta X aplicada al “Software Testing” de servicios web.
+•	Toda funcionalidad del sistema y transacción de negocio debe responder al usuario en menos de 5 segundos.
+•	El sistema debe ser capaz de operar adecuadamente con hasta 100.000 usuarios con sesiones concurrentes.
```
Nada de calidad del equipo, solo de servicios

ricardo-andre-pool-cen commented on Act#4 Requisitos no funcionales:
```diff
@@ -1,2 +1,42 @@
+•	Toda funcionalidad del sistema y transacción de negocio debe responder al usuario en menos de 5 segundos.
+•	El sistema debe ser capaz de operar adecuadamente con hasta 100.000 usuarios con sesiones concurrentes.
+•	Los datos modificados en la base de datos deben ser actualizados para todos los usuarios que acceden en menos de 2 segundos
+•	Los permisos de acceso al sistema podrán ser cambiados solamente por el administrador de acceso a datos.
```
Estos requisitos si pueden ir

ricardo-andre-pool-cen commented on Act#4 Requisitos no funcionales:
```diff
@@ -1,2 +1,42 @@
+•	El nuevo sistema debe desarrollarse aplicando patrones y recomendaciones de programación que incrementen la seguridad de los datos.
+•	Todos los sistemas deben respaldarse cada 24 horas. Los respaldos deben ser almacenados en una localidad segura ubicada en un edificio distinto al que reside el sistema.
+•	Todas las comunicaciones externas entre servidores de datos, aplicación y cliente del sistema deben estar encriptadas.
+•	Si se identifican ataques de seguridad o brecha del sistema, el mismo no continuará operando hasta ser desbloqueado por un administrador de seguridad.
```
Tampoco irían prevenciones ante ataques, no sabemos como esta compuesto el equipo

ricardo-andre-pool-cen commented on Act#4 Requisitos no funcionales:
```diff
@@ -1,2 +1,42 @@
+•	El tiempo para iniciar o reiniciar el sistema no podrá ser mayor a 5 minutos.
+•	La tasa de tiempos de falla del sistema no podrá ser mayor a X% del tiempo de operación total.
+•	El promedio de duración de fallas no podrá ser mayor a una tasa considerada por la ley.
+•	La probabilidad de falla del Sistema no podrá ser mayor a una X tasa.
```
No se pueden calcular tasas de error

ricardo-andre-pool-cen commented on Act#4 Requisitos no funcionales:
```diff
@@ -1,2 +1,42 @@
+•	Sistemas de datos médicos: El nuevo sistema y sus procedimientos de mantenimiento de datos deben cumplir con las leyes y reglamentos de protección de datos médicos.
+•	El nuevo sistema se acogerá a las reglas de las licencias generales públicas (GNU), es decir será gratuito, código abierto en el que cualquiera podrá cambiar el software, sin patentes y sin garantías.
+•	Las páginas web a ser desarrolladas deben cumplir con la ley de tratamiento en condiciones de igualdad para personas con discapacidad.
+•	El sistema no revelara a sus operadores otros datos personales de los clientes distintos a nombres y números de referencia.
```
Esto si puede entrar

ricardo-andre-pool-cen commented on Prueba pull request:
```diff
@@ -0,0 +1 @@
+request
```
Borra este documento

ricardo-andre-pool-cen commented on Requisitos funcionales:
```diff
@@ -0,0 +1,82 @@
+Requisitos funcionales.
+•	El sistema enviará un mensaje cuando se registre alguna de las siguientes transacciones: compra del cliente de alguna mensualidad o algún posible “Student Development Package”, anulación o suscripción de posibles mensualidades o “recursos premium”, emisión de factura a cliente y registro de pago de cliente.
+•	Al aprobar un pedido, la solicitud pasará al siguiente paso del flujo de trabajo (workflow) de aprobación configurado en el sistema.
+•	El sistema permitirá a los usuarios autorizados el ingresar planes y cronogramas de proyecto.
```
Casi todos los requisitos funcionales pueden ir, ten en cuenta las observaciones que te dije con los no funcionales, aplican tambien.

ricardo-andre-pool-cen reviewed (changes requested):
Recuerda que estos son ejemplos, tenemos que construir nuestros requisitos

LuisGilDzib committed:
Update Requisitos funcionales

He realizado la supervisión y edición de los requisitos para que sean útiles para nuestro proyecto, igual se ha agregado el apartado de excepciones con su respectivo contenido.

LuisGilDzib committed:
Delete Prueba pull request

LuisGilDzib committed:
Update Act#4 Requisitos no funcionales

He actualizado los requisitos no funcionales.
